src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/page/customfooterlibs.html

```diff
@@ -29,3 +29,11 @@
 ></sly>
 #end
 
+#if ($includeForms == "y" or $includeFormsenrollment == "y" or $includeFormscommunications == "y" or $includeFormsheadless == "y")
```

Comment:
we should include runtime clientlibs in [xfpage](https://github.com/adobe/aem-project-archetype/tree/develop/src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/xfpage) as well.

Reply:
+1

Reply:
Done